packages/runtime/src/next/api-handler/handlers/redirect-preview.ts

```diff
@@ -51,30 +51,37 @@ export default async function redirectPreviewHandler(
 async function redirectPreviewRouteHandler(
   _request: NextRequest,
   _context: Context,
-  {}: { apiKey: string },
+  { }: { apiKey: string },
```

Comment:
This looks like maybe something Prettier should've caught?

Reply:
It *is* a Prettier reformat.

Reply:
We don't run Prettier checks on CI.

Reply:
We probably should 😄